source/clear-linux/concepts/telemetry-about.rst

```diff
@@ -0,0 +1,76 @@
+.. _telemetry-about:
+
+Telemetrics
+###########
+
+Telemetry is one of the key features of |CLOSIA|. The |CL| team is focused on
+the quality of the telemetry data, and is moving from issues that are detected
```

Comment:
@MCamp859. I struggle to understand the intent of this sentence, especially after "...is moving from issues that are detected..." Can you revise this to be a little more user-centric? For example, consider: What might a Linux Developer most want to know about how default telemetry benefits them?

Reply:
So the point here is that instead of end users discovering bugs and reporting them (i.e. reactive), the CL team is able to monitor system health and proactively fix issues that show up in the telemetry data before an end user encounters them.  I agree that it could be worded differently